data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
+      [ "Хаус", "House" ],
+      [ "Доти", "Doty" ],
+      [ "Драйвер", "Driver" ],
+      [ "Дрейк", "Drake" ],
+      [ "Дрейпер", "Draper" ],
+      [ "Френд", "Friend" ],
+      [ "Дуайер", "Dwyer" ],
+      [ "Дюбе", "Dube" ],
+      [ "Дусетт", "Doucette" ],
+      [ "Дэйвид", "David" ],
+      [ "Дэйгл", "Daigle" ],
+      [ "Дэйли", "Dailey" ],
+      [ "Дюбоз", "Dubose" ],
+      [ "Дюк", "Duke" ],
+      [ "Дюкс", "Dukes" ],
+      [ "Дюпри", "Dupree" ],
+      [ "Бишоп", "Bishop" ],
+      [ "Жан", "Jean" ],
+      [ "Вест", "Vest" ],
+      [ "Лайт", "Light" ],
+      [ "Лоу", "Law" ],
+      [ "Лоз", "Laws" ],
+      [ "Сток", "Stock" ],
+      [ "Зейглер", "Zeigler" ],
+      [ "Зейтц", "Seitz" ],
+      [ "Зепеда", "Zepeda" ],
+      [ "Винтер", "Winter" ],
+      [ "Винтерс", "Winters" ],
+      [ "Голд", "Gold" ],
+      [ "Ингланд", "England" ],
+      [ "Иризарри", "Irizarry" ],
+      [ "Айрленд", "Ireland" ],
+      [ "Спаркс", "Sparks" ],
+      [ "Йе", "Yee" ],
+      [ "Йейтс", "Yates" ],
+      [ "Кавасос", "Cavazos" ],
+      [ "Казарес", "Cazares" ],
+      [ "Хай", "High" ],
+      [ "Стоун", "Stone" ],
+      [ "Рид", "Reed" ],
+      [ "Карбахал", "Carbajal" ],
+      [ "Карвалью", "Carvalho" ],
+      [ "Кардуэлл", "Cardwell" ],
+      [ "Карриер", "Carrier" ],
+      [ "Кард", "Card" ],
+      [ "Кастл", "Castle" ],
+      [ "Кафлин", "Coughlin" ],
+      [ "Кейгл", "Cagle" ],
+      [ "Келер", "Koehler" ],
+      [ "Келлихер", "Kelleher" ],
+      [ "Келлогг", "Kellogg" ],
+      [ "Кениг", "Koenig" ],
+      [ "Керли", "Curley" ],
+      [ "Кимбро", "Kimbrough" ],
+      [ "Кисер", "Kiser" ],
+      [ "Кифе", "Keefe" ],
+      [ "Лей", "Lay" ],
+      [ "Клемонс", "Clemons" ],
+      [ "Клейн", "Klein" ],
+      [ "Кнутсон", "Knutson" ],
+      [ "Коварт", "Cowart" ],
+      [ "Кодилл", "Caudill" ],
+      [ "Коксуэйн", "Coxwain" ],
+      [ "Колер", "Kohler" ],
+      [ "Коллэвей", "Callaway" ],
+      [ "Колтер", "Coulter" ],
+      [ "Кольер", "Collier" ],
+      [ "Комо", "Comeau" ],
+      [ "Конвей", "Conway" ],
+      [ "Конн", "Conn" ],
+      [ "Спир", "Spear" ],
+      [ "Рут", "Root" ],
+      [ "Кинг", "King" ],
+      [ "Коррал", "Corral" ],
+      [ "Корреиа", "Correia" ],
+      [ "Кот", "Cote" ],
+      [ "Коффман", "Coffman" ],
+      [ "Эдж", "Edge" ],
+      [ "Крауз", "Crouse" ],
+      [ "Мик", "Meek" ],
+      [ "Винг", "Wing" ],
+      [ "Крэбтри", "Crabtree" ],
+      [ "Крэндалл", "Crandall" ],
+      [ "Крювс", "Crews" ],
+      [ "Хук", "Hook" ],
+      [ "Куоллс", "Qualls" ],
+      [ "Барроу", "Barrow" ],
+      [ "Курран", "Curran" ],
+      [ "Кухн", "Kuhn" ],
+      [ "Стек", "Stack" ],
+      [ "Кэппс", "Capps" ],
+      [ "Кэрон", "Caron" ],
+      [ "Кэррингтон", "Carrington" ],
+      [ "Лавуа", "Lavoie" ],
+      [ "Лакки", "Lackey" ],
+      [ "Ланкфорд", "Lankford" ],
+      [ "Лансфорд", "Lunsford" ],
+      [ "Ларю", "Larue" ],
+      [ "Леал", "Leal" ],
```

Comment:
Just as a note: I've checked this, and it seems to be a Spanish surname, which would probably be pronounced `Леал`, rather than `Лил` if it were English.